test/framework/bootstrap/kind_provider.go

```diff
@@ -29,8 +29,11 @@ import (
 )
 
 const (
-	// DefaultNodeImage is the default node image to be used for for testing.
-	DefaultNodeImage = "kindest/node:v1.22.0"
+	// DefaultNodeImageRepository is the default node image repository to be used for testing.
+	DefaultNodeImageRepository = "kindest/node"
+
+	// DefaultNodeImage is the default node image to be used for testing.
+	DefaultNodeImage = DefaultNodeImageRepository + ":v1.22.0"
```

Comment:
```suggestion
	// DefaultNodeImage is the default Kubernetes version to be used for creating a kind cluster.
	DefaultNodeImageVersion = "v1.22.0"
```
So we are tracking elementary infos and we delegate composing them to the code